meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -254,40 +254,27 @@ do_install() {
         ${@bb.utils.contains('PACKAGECONFIG', 'modules', 'install-modules', '', d)}
 
     # install inventory scripts
-    if grep -q install-inventory-local-scripts ${B}/src/${GO_IMPORT}/Makefile; then
-        oe_runmake \
-            -C ${B}/src/${GO_IMPORT} \
-            V=1 \
-            prefix=${D} \
-            datadir=${datadir} \
-            install-inventory-local-scripts \
-            ${@bb.utils.contains('PACKAGECONFIG', 'inventory-network-scripts', 'install-inventory-network-scripts', '', d)}
-    else
+    oe_runmake \
+        -C ${B}/src/${GO_IMPORT} \
+        V=1 \
+        prefix=${D} \
+        datadir=${datadir} \
+        install-inventory-local-scripts \
+        ${@bb.utils.contains('PACKAGECONFIG', 'inventory-network-scripts', 'install-inventory-network-scripts', '', d)}
+
+    # install dbus files
+    if ${@bb.utils.contains('PACKAGECONFIG', 'dbus', 'true', 'false', d)}; then
+        # install the D-Bus policy file
```

Comment:
This part will soon go away as well, see https://github.com/mendersoftware/meta-mender/commit/fb1159a85c37e713a46ed6f7be9a5e5396871dff